packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fixed packaging of daemon binary for macOS. [#2126](https://github.com/zowe/zowe-cli/pull/2126)
```

Comment:
Can we explain what the bug was, like what was happening that should not have been happening?